openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -338,6 +338,24 @@ export default function init(){
             $('.wmd-preview').before('<h3 id="prevHead" style="margin:15px 0 10px;padding:0;">Preview</h3>');
         }
     });
+
+    // For Book Preview Button
+    $(document).ready(function(){
+        $('#bookPreviewButton').colorbox({
+            inline: true,
+            opacity: '0.5',
+            href: '#bookPreview'
+        })
+    })
+
+    $('.lazyIframe').show(function(){
```

Comment:
https://api.jquery.com/show/ doesn't take a function as a parameter.  Chain instead?

Reply:
Really ?? Almost all the examples I see in those very docs take a function() as a parameter. Happy to change it if necessary

Reply:
I think changing it would best; since it's not one of the signatures on the API page, it could be deprecated without warning (it was also hard for me to figure out what this does cause I didn't see its signature on the API page, so readability improves).

Reply:
🤦‍♂ Nevermind! You're completely right, it's the second example!
![image](https://user-images.githubusercontent.com/6251786/64354443-72a25f80-cfcd-11e9-8ce5-22247c7dd8f1.png)

I was misreading the `[]`. This is good as is :+1: